Execution of io_test (bash_tests.testcases.TestCases) fails on Windows 11

When I run the tests with python3 runner.py on my local Windows 11 system, it fails with io_test:

```
ERROR: io_test (bash_tests.testcases.TestCases)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "C:\GitHub\norbertwg_exiv2\tests\bash_tests\testcases.py", line 595, in io_test
    out     += BT.Executer('iotest ' + ' '.join(files))
  File "C:\GitHub\norbertwg_exiv2\tests\bash_tests\utils.py", line 488, in __init__
    self.run()
  File "C:\GitHub\norbertwg_exiv2\tests\bash_tests\utils.py", line 527, in run
    raise RuntimeError('\n' + log.to_str())
RuntimeError:
[INFO] Starting HTTP server ...
[INFO] The HTTP server started
[ERROR] Failed to execute: C:\GitHub\norbertwg_exiv2\build\bin\iotest s0 s1 s2 http://127.0.0.1:12760/table.jpg
[ERROR] The asserted return code is [0], but got 1
[INFO] OUTPUT:
error - unable to connect to server = 127.0.0.1 port = 12760 wsa_error = 0
Caught Exiv2 exception 'http: Failed to open file (-1): /table.jpg'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\GitHub\norbertwg_exiv2\tests\bash_tests\testcases.py", line 605, in io_test
    raise RuntimeError('\n' + BT.log.to_str())
RuntimeError:
[INFO] Starting HTTP server ...
[INFO] The HTTP server started
[ERROR] Failed to execute: C:\GitHub\norbertwg_exiv2\build\bin\iotest s0 s1 s2 http://127.0.0.1:12760/table.jpg
[ERROR] The asserted return code is [0], but got 1
[INFO] OUTPUT:
error - unable to connect to server = 127.0.0.1 port = 12760 wsa_error = 0
Caught Exiv2 exception 'http: Failed to open file (-1): /table.jpg'
[ERROR]
[INFO] Starting HTTP server ...
[INFO] The HTTP server started
[ERROR] Failed to execute: C:\GitHub\norbertwg_exiv2\build\bin\iotest s0 s1 s2 http://127.0.0.1:12760/table.jpg
[ERROR] The asserted return code is [0], but got 1
[INFO] OUTPUT:
error - unable to connect to server = 127.0.0.1 port = 12760 wsa_error = 0
Caught Exiv2 exception 'http: Failed to open file (-1): /table.jpg'
```

This has a lower priority. As this error occurs even with the original sources from GitHub I ignore it when I test my changes. But I would like to understand it and avoiding this makes it more comfortable to verify my changes.

--- a/src/http.cpp
+++ b/src/http.cpp
@@ -202,7 +202,7 @@ int Exiv2::http(Exiv2::Dictionary& request, Exiv2::Dictionary& response, std::st
 
   // convert unknown servername into IP address
   // http://publib.boulder.ibm.com/infocenter/iseries/v5r3/index.jsp?topic=/rzab6/rzab6uafinet.htm
-  if (inet_pton(AF_INET, servername_p, &serv_addr.sin_addr) <= 0) {
+  if (inet_pton(AF_INET, servername_p, &serv_addr.sin_addr) != 0) {
     struct addrinfo hints = {};
     hints.ai_family = AF_INET;
     hints.ai_socktype = SOCK_STREAM;
@@ -226,8 +226,8 @@ int Exiv2::http(Exiv2::Dictionary& request, Exiv2::Dictionary& response, std::st
   server = connect(sockfd, reinterpret_cast<const sockaddr*>(&serv_addr), serv_len);
   if (server == SOCKET_ERROR && WSAGetLastError() != WSAEWOULDBLOCK) {
     closesocket(sockfd);
-    return error(errors, "error - unable to connect to server = %s port = %s wsa_error = %d", servername_p, port_p,
-                 WSAGetLastError());
+    return error(errors, "error - unable to connect to server = %s port = %s wsa_error = %d", servername_p,
+                 std::to_string(serv_addr.sin_port).c_str(), WSAGetLastError());
   }
 
   char buffer[32 * 1024 + 1];

